src/Semantic/Graph.hs

```diff
@@ -301,36 +299,32 @@ parsePythonPackage parser project = do
       resMap <- Task.resolutionMap project
       pure (Package.fromModules (Data.Abstract.Evaluatable.name (projectName project)) modules resMap) -- TODO: Confirm this is the right `name`.
     PythonPackage.Packages dirs -> do
-      filteredBlobs <- for dirs $ \dir -> do
-        let packageDir = projectRootDir project </> unpack dir
-        let paths = filter ((packageDir `isPrefixOf`) . filePath) (projectFiles project)
-        traverse (readFile project) paths
+      let filteredBlobs = do
+            dir <- dirs
+            let packageDir = projectRootDir project </> unpack dir
+            filter ((packageDir `isPrefixOf`) . blobPath) (projectBlobs project)
       packageFromProject project filteredBlobs
     PythonPackage.FindPackages excludeDirs -> do
       trace "In Graph.FindPackages"
       let initFiles = filter (("__init__.py" `isSuffixOf`) . filePath) (projectFiles project)
       let packageDirs = filter (`notElem` ((projectRootDir project </>) . unpack <$> excludeDirs)) (takeDirectory . filePath <$> initFiles)
-      filteredBlobs <- for packageDirs $ \dir -> do
-        let paths = filter ((dir `isPrefixOf`) . filePath) (projectFiles project)
-        traverse (readFile project) paths
+      let filteredBlobs = do
```

Comment:
Stylistic nit, feel free to ignore: can we combine these `let` bindings? I.e. instead of

```haskell
let initFiles = …
let packageDirs = …
let filteredBlobs = …
```

do something like

```haskell
let initFiles = …
    packageDirs = …
    filteredBlobs = …
```